refactor: Zustand 상태 업데이트 네이밍 및 함수 작성 스타일 통일

faber-h opened the pull request:
### ✨ 이슈 번호

- #162

### 📌 설명

- `Zustand` 상태 업데이트 함수들의 `set` 호출 방식을 일관되게 통일합니다.
- 예: `setPrevChannelId: (channelId) => set({ prevChannelId: channelId })` 형태를
`setPrevChannelId: (prevChannelId) => set({ prevChannelId })`로 변경합니다.
- 상태 `setter` 함수들의 파라미터명을 상태명과 동일하게 통일합니다
- `Zustand` 스타일을 통일하여 작성한 Pull Request입니다.

### 📃 작업 사항

- [x] `set({ key: value })` 형태를 `set({ key })`로 단순화
- [x] 상태 `setter` 함수들의 파라미터명을 상태명과 동일하게 통일

### 💭 리뷰 사항 반영

### ✅ Pull Request 체크 사항

- [x] 가장 최신 브랜치를 pull 하였습니다.
- [x] base 브랜치명을 확인하였습니다.
- [x] 코드 컨벤션을 모두 확인하였습니다.
- [x] 브랜치명을 확인하였습니다.

faber-h committed:
refactor: `set({ key: value })` → `set({ key })` 형태로 간결하게 리팩토링

faber-h committed:
refactor: 상태 setter 함수들의 파라미터명을 상태명과 동일하게 통일

d0-neii commented on src/User/hooks/useUpdateSetting.jsx:
```diff
@@ -3,7 +3,7 @@ import { updateUserSettings } from "@/User/services/userSettings";
 
 const useUpdateSetting = (type) => {
-  const { settings, setUserSettings } = useUserSettingsStore();
+  const { settings, setSettings } = useUserSettingsStore();
```
`setSettings`를 `setUserSettings`처럼 이름을 좀 더 명확하게 적어주면 좋을 것 같아요!
`setUserSettings`라고 하면 어떤 값인지 더 직관적으로 이해할 수 있을 것 같습니다:)

@faber-h, @devjaeyoon 두 분 생각도 궁금합니다~

devjaeyoon replied:
@Eun0713 좋은 의견이라고 생각합니다!

`setSettings`보다 `setUserSettings`가 사용자 설정이라는 기능에 더 적합한 변수명인 것 같아요.
`setUserSettings`를 적용하면 `settings` 또한 `userSettings`로 변경해야겠네요!

faber-h replied:
이름이 코드의 목적을 드러내도록 변수를 리팩토링하자는 두 분 의견에 공감해요!
`setUserSettings`는 변경 전으로 되돌리고
`settings`는 `userSettings`로 수정하겠습니다!

faber-h committed:
refactor: `settings` → `userSettings`로 네이밍 명확화

- `settings`와 `setSettings`를 `userSettings`, `setUserSettings`로 이름 변경
- 네이밍 충돌 방지 및 의미 명확화를 위한 리팩토링

devjaeyoon reviewed (approved):
고생하셨습니다~ LGTM!

Eun0713 reviewed (approved):
LGTM🍀
